docs/global.css

```diff
@@ -98,13 +98,24 @@
   color: #2f2d44;
 }
 
+:global body h5 {
```

Comment:
This whole file is quite repetitive.

Can you have a shared block for all the `h*`s which contains:
```
  font-family: "Poppins";
  font-weight: 500;
  letter-spacing: -0.02em;
  color: #2f2d44;
```

and then apply custom styling for each of them